src/main/java/org/openpnp/gui/VisionSettingsPanel.java

```diff
@@ -0,0 +1,203 @@
+package org.openpnp.gui;
+
+import org.openpnp.gui.components.AutoSelectTextTable;
+import org.openpnp.gui.support.*;
+import org.openpnp.gui.tablemodel.VisionSettingsTableModel;
+import org.openpnp.model.*;
+import org.openpnp.model.Package;
+
+import javax.swing.*;
+import javax.swing.table.TableRowSorter;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.prefs.Preferences;
+import java.util.stream.Collectors;
+
+public class VisionSettingsPanel extends JPanel implements WizardContainer {
+
```

Comment:
Should this extend AbstractConfigurationWizard, like all the other Wizards? 
(Note, I don't actually know this stuff very well, I just like things consistent).

Reply:
`VisionSettingsPanel` follows the same pattern as other panels (Parts, Packages, ... ) in the `gui `folder. `BottomVisionSettingsConfigurationWizard `extends `AbstractConfigurationWizard` as it is the settings panel that uses the `Apply `and `Reset `buttons.

Reply:
Ah ok!